Gopkg.toml

```diff
@@ -10,4 +10,4 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "release/2.11"
```

Comment:
@AlekSi Should I revert it to PMM-2.0? Thanks.

Reply:
yes